s12/protobuf/proto/validator_helpers.go

```diff
@@ -34,9 +34,9 @@ func IsLegacyID(str string) bool {
 	return rxLegacyId.MatchString(str)
 }
 
-// IsValidEmail checks if an email address is a valid RFC 5322 address
+// IsValidEmail checks if an email address is a valid email address that can be delivered
 func IsValidEmail(str string) bool {
-	return govalidator.IsEmail(str)
+	return govalidator.IsExistingEmail(str)
```

Comment:
As discussed, `IsExistingEmail` cannot replace `IsEmail` as the validation in that method is weak. Please refer to how this has been implemented here: https://github.com/SafetyCulture/s12-proto-private/blob/f45338bf6b61f4e9ee3d201fb85e976ecff5b163/s12/protobuf/proto/validator_helpers.go#L37
Suggest to not use this type of validation at this point though.

Reply:
@AipengChen, do we need to update the PR to align with Peter's comment?